pkg/config/config_test.go

```diff
@@ -931,23 +931,25 @@ func TestNewConfig(t *testing.T) {
 					ImageTag:      "init",
 					AllowedImages: []string{"image"},
 					AllowedTags:   []string{"tag", "tag2"},
-					RequiredEdges: map[string]string{
-						SpiceDBMigrationState{
-							Tag: "init",
-						}.String(): SpiceDBMigrationState{
-							Tag: "tag",
-						}.String(),
-					},
-					Nodes: map[string]SpiceDBMigrationState{
-						SpiceDBMigrationState{
-							Tag: "init",
-						}.String(): {
-							Tag: "init",
+					UpdateGraph: UpdateGraph{
+						RequiredEdges: map[string]string{
+							SpiceDBMigrationState{
+								Tag: "init",
+							}.String(): SpiceDBMigrationState{
+								Tag: "tag",
+							}.String(),
```

Comment:
a helper func here could go a long way towards readability

Reply:
refactored to used the graph builder helpers